janet/src/main/java/io/techery/janet/helper/ActionStateSubscriber.java

```diff
@@ -29,6 +33,11 @@
         return this;
     }
 
+    public ActionStateSubscriber<A> onFinish(Action0 onFinish) {
```

Comment:
I see no case why it couldn't be helpful and consistent. Sometime we need action to act as an identifier -> in this case we need action at onFinish. Imagine, you're showing list of downloads, there a bunch of same actions and you have to identify which download to show as stopped on finish. Etc.